Assignment1/Activity2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>​
+<Table>​
+  <Title>
+    <Title1>Empno</Title1>
+    <Title2>Empname</Title2>
+    <Title3>Salary</Title3>
+  </Title>
```

Comment:
should have been used as element names for further rows (instead of Field1, Field2,...)